firmware/application/ui/ui_spectrum.cpp

```diff
@@ -390,19 +390,23 @@ uint32_t filter_bandwidth_for_sampling_rate(int32_t sampling_rate) {
                                        // BW 500k...750kHz, ex. 500kHz (fs = 8 x BW = 4Mhz), BW 600kHz (fs = 4,8Mhz), BW 750 kHz (fs = 6Mhz).
             return 2'500'000;          // In some IC, MAX2837 appears as 2250000, but both work similarly.
 
-        case 8'800'000:  // BW capture 1,1Mhz fs = 8 x 1,1Mhz = 8,8Mhz. (1Mhz showed slightly higher noise background).
-            return 3'500'000;
+        case 8'000'000:        // BW capture 1Mhz fs = 8 x 1Mhz = 8Mhz. (1Mhz showed slightly higher noise background).
+            return 3'500'000;  // some low SD cards, if not showing avg. writting speed >4MB/sec, they will produce sammples drop at REC with 1MB and C16 format.
+
+        case 12'000'000:  // BW capture 1,5Mhz, fs = 8 x 1,5Mhz = 12Mhz
```

Comment:
I don't know yet if it's Capture or Replay, but 1.5MHz and higher aren't playing back correctly.  I'll provide more test results below.

Reply:
Well in the wiki ,  we already mentioned that the Yellow icon is not reliable at all to be replayed . I is only usefull to see the waterfall spectrum .  (see current wiki) 
![image](https://github.com/eried/portapack-mayhem/assets/86470699/303f4bee-8a8a-4466-aa1f-c31c02cf5af7)

But I have to agree with you ,  that 1M5 and above can only be used to see on the screen waterfall , not to REC. 
Then maybe we should hide the icon instead of yellow or blocking now in current version by SW  the REC feautre in those NG options.  Or this can also be done later in next PR, Up to us . 
.

Reply:
Looks like 1.5MHz capture rates are only writing half of the samples to the capture file.  It's not an SD card write performance limit.  (I think we should figure out why instead of disabling it.)